frappe/utils/boilerplate.py

```diff
@@ -278,6 +278,31 @@ def make_boilerplate(dest, app_name):
 #
 # auto_cancel_exempted_doctypes = ["Auto Repeat"]
 
+
+# User Data Protection
```

Comment:
Should this happen via Doctype configurations instead? That might make it easier for users to adapt to policy changes as well.

Reply:
Yeah. The thought is to have Data Protection Settings at a docfield level. But this may need to have more iterations to get there. For now, I'm treating this refactor as the first step towards management via configurations.